frontend/src/metabase/entities/bookmarks.js

```diff
@@ -23,7 +23,18 @@ const Bookmarks = createEntity({
   objectSelectors: {
     getIcon,
   },
-
+  objectActions: {
```

Comment:
@ranquild 

Thank you.

Updated and I've added a couple of lines of comments in `lib/entities` with what you wrote above.
Maybe it can be improved, but I think it's better to have something than nothing.